### DISCUSSION TITLE
Netflix 想顛覆串流媒體評分系統

### DISCUSSION THREAD

**Discussion by Jay-722** [1101-GS4719-C/Python-Community-GS4719-C#564] [General]
為什麼要在熟悉的讚和不讚圖案外增加「雙讚」呢？據 Netflix 推薦演算法，用戶點讚或不讚，會得到更多客製推薦，但以用戶回饋看，讚和不讚已不夠精準，希望讓「喜歡」和「更喜歡」有區隔，讓 Netflix 推薦能更精確。

「雙讚」就是 Netflix 想到的方法，有「微調」演算法的作用。讚仍是表達「喜歡」什麼，Netflix 也會[據觀看紀錄或方式時間推薦]。但「雙讚」不同。用戶告訴 Netflix 對特定內容情有獨鍾，包括演員、製作團隊、類型、小眾與否等，Netflix 推薦就能更具體。

多了這個雙讚，當我真的很喜歡一部劇或是一部影片的時候，他就可以推薦我相似的劇或影片來看，我個人是覺得不錯的，現在不管是intergram或是facebook等等都會藉由我們平常看了什麼，或對什麼東西有興趣，都會有相關的廣告跑出來，但我覺得這個稍微恐怖，感覺隱私有點不安全。

全文:https://technews.tw/2022/04/25/netflix-3-stars/

**Comment by PoooYen** [comment]
簡單來說就是把喜歡的程度分級對吧

**Comment by Sazmafuji** [comment]
這系統真棒，每次只能按讚而沒辦法按超讚，雖然我覺得星星就能夠去區別評分，但超讚也不錯

**Comment by 11timyyy** [comment]
youtube的演算法也跟這個機制有點相像，有時還會跑出幾年前的事物XD

**Comment by ghost** [comment]
這個跟隱私也沒什麼太直接的關係吧
要我們的隱私有太多方法了XD

**Comment by bloodtrail0511** [comment]
好辦法，把二元對立的評分方法階層化，演算法的推送就能更精準

**Comment by ThomasMeng0515** [comment]
希望這個方式會拯救Netflix的股價哈哈

**Comment by yucheng1028** [comment]
現在的網路感覺早就沒有隱私了
所以我不覺得有隱私上的問題2ㄏ

**Comment by leon10456** [comment]
感謝分享 現在能點超喜歡了

**Comment by HuanYuuuuuu** [comment]
雙讚你的這篇貼文

**Comment by Ku-Hao-Chen** [comment]
謝謝分享，這個感覺很不錯。

**Comment by henrywen547** [comment]
雖然不知道現在對大眾來說有多少效果，只是以我而言可能還是不喜歡按雙讚啦

**Comment by etin2468** [comment]
有多的一種評分方式可以讓影片的評價更明確，感覺不錯ㄟ

**Comment by d26488399** [comment]
感覺一個讚就夠了，多一個反而不知道甚麼時候要按

**Comment by JSHT** [comment]
覺得不如改為評分制來的更加直觀，這樣也可以將喜歡、更喜歡分隔。

**Comment by Yuhsin000** [comment]
若能因此增進用戶體驗，我想大家會願意順手按一下的～～

**Comment by Lai27330** [comment]
感覺上聽起來是比目前的五星評分制度來的更為準確，但是對我來說大部分選擇影集只是出於當時的感覺，並不會像音樂一樣有固定的偏好，也因此不會特別去按讚或是搜尋相關的影片，所以對我這種什麼都看一點的人來說好像反而沒有太大的幫助。

**Comment by cordydeng** [comment]
我覺得雙讚如果能和推薦內容機制搭配一定有效，例如一個人喜歡愛情喜劇，就推薦他雙讚數多的愛情喜劇之類的

**Comment by Rosancl** [comment]
推這個功能，Netflix目前的適合觀看的程度分析對我來說已經有一定的參考價值，有雙讚的話想必會更加準確。

**Comment by yenwei12138** [comment]
大家都會去按喜歡或讚!?我不管是看Netflix或Youtube都不太習慣按讚，還以為其實都沒什麼人在按

**Comment by yi-1111** [comment]
感覺跟youtube的演算法也機制有點相像

**Comment by liangyinglly** [comment]
我覺得可以給一些形容詞給大家選擇點擊，總覺得讚與不讚，喜歡與不喜歡太極端了，有時候電影或影集真的很看每個人的喜好

**Comment by Ben6uo** [comment]
所以是跟youtube差不多吧?都是跟點擊和搜尋次數有關

**Comment by Jimmy911218** [comment]
這樣確實可以更接近自己想看的東西，但就我而言我並不喜歡一只看抹些東西這機制反而限制了我

**Comment by leon10456** [comment]
感謝樓主分享

**Comment by 110409538** [comment]
有時候點讚並不是劇有多好看,就只是為了支持某個演員而已.

**Comment by HaoFish123** [comment]
感謝分享，但我還是覺得這樣分沒啥意義。

**Comment by treeann** [comment]
前陣子才停止訂閱netflix

**Comment by stan890425** [comment]
Netflix 不適本來就有演算法了嗎?

**Comment by jcpan821** [comment]
感謝分享~~

**Comment by alexthebigpp** [comment]
感覺蠻有趣的，希望也可以出像youtube那樣倒讚的功能，前提是要看的見哈哈

**Comment by Lynn12378** [comment]
感謝分享，感覺這樣就能被推薦更多我可能比較感興趣的影集了
